docs/Hooks.md

```diff
@@ -77,6 +77,8 @@ fastify.addHook('onResponse', async (res) => {
 })
 ```
 
+If the `async`/`await` form is used, (or equivalently, if the hook returns a `Promise`), be sure to not call `next`. Otherwise you will duplicate the execution of any further handlers.
```

Comment:
How about this instead:

```md
> ### Notice: the `next` callback is not available when using `async`/`await` or returning a `Promise`. If you do invoke a `next` callback in this situation unexpected behavior may occur, e.g. duplicate invocation of handlers.
```

Reply:
Changed! I did however swap the `h3` for just a bold prefix, which is a bit more readable.

Reply:
The `> ` was key to the formatting with the subheading markup. I'm fine with what you have though.

Reply:
Learning every day!